### PR TITLE
Build v 0.6.3:

### PR DESCRIPTION
**Pull request by akabanovs**
- Version set to 0.6.3 and sha256 updated.
 - Patches added.
 - Build number reset to 0.
 - Build script updated.
 - Dependencies revised and redundant ones removed.
 - enhanced tests with pip check.
 - 'about' section enhanced and urls updated.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,61 +1,47 @@
...
+    - setuptools
     - pip
-    - numpy
+    - wheel
+    - numpy  {{ numpy  }}
+    - libosqp {{ version }}
   run:
     - python
     - {{ pin_compatible('numpy') }}
+    - numpy >=1.7
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:42]: Will this `numpy` duplication cause issues in the runtime?

Comment (akabanovs) [reply recipe/meta.yaml:42]: No, I don't think so. This just applies two constraints, compatible version but not less than 1.7.  pin_compatible also applies upper limit (2.0).

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**